README.md

```diff
@@ -5,7 +5,7 @@
 
 <img src="./media/sindri-gradient-logo.webp" height="160" align="right"/>
 
-#### [Sindri Sign Up](https://sindri.app/signup) | [Scroll SDK Docs](https://scroll-sdk-init.docs.scroll.xyz/en/sdk/) | [Getting Started](#getting-started) | [Development](#internal-development)
+#### [Sindri Sign Up](https://sindri.app/signup) | [Scroll SDK Docs](https://docs.scroll.io/en/sdk) | [Getting Started](#getting-started) | [Development](#internal-development)
```

Comment:
```suggestion
#### [Sindri Sign Up](https://sindri.app/signup) | [Scroll SDK Docs](https://docs.scroll.io/en/sdk/) | [Getting Started](#getting-started) | [Development](#internal-development)
```

Reply:
Apparently Scroll wants a trailing slash and appends one if it's not on the URL. This change simply adds the trailing slash for consistency (see new link further down in this PR).